addons/transform/org.openhab.transform.feed.test/src/test/resources/output/rss_1.0.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns="http://purl.org/rss/1.0/" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:taxo="http://purl.org/rss/1.0/modules/taxonomy/">
+  <channel rdf:about="http://www.feedforall.com">
+    <title>Sample Feed</title>
```

Comment:
@kaikreuzer is about the licenses, I have some feelings about it but not yet a clear view.